File: src/frontend/src/app/components/atendimento/convenio/model/codigo-convenio-dto.ts
import { Exclude, Expose } from 'class-transformer';

@Exclude()
export class CodigoConvenioDTO {
  @Expose() id?: string;
  @Expose() convenioId?: string;
  @Expose() procedimentoId?: string;
  @Expose() procedimentoCodigo?: string;
  @Expose() procedimentoDescricao?: string;
  @Expose() codigo?: string;
}

File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/atendimento/codigoconvenio/dto/CodigoConvenioDTO.java
package br.com.grupopipa.gestaointegrada.atendimento.codigoconvenio.dto;

import java.util.UUID;

import br.com.grupopipa.gestaointegrada.core.dto.DTO;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class CodigoConvenioDTO implements DTO {

    private UUID id;
    private UUID convenioId;
    private UUID procedimentoId;
    private String procedimentoCodigo;
    private String procedimentoDescricao;
    private String codigo;
}
